playwright.config.ts

```diff
@@ -36,7 +36,7 @@ export default defineConfig({
     : [["html"], ["list"]],
   expect: { timeout: 10 * 1000 },
   maxFailures: 10,
-  timeout: 30 * 1000,
+  timeout: 35 * 1000,
```

Comment:
Suggestion: I think we should break down "long" tests into smaller tests if that's possible.

Reply:
Totally agree, let's first stabilize them and then refactor the 'long' one